src/test/ui/suggestions/dont-suggest-ufcs-for-const.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0599]: no method named `MAX` found for type `u32` in the current scope
+  --> $DIR/dont-suggest-ufcs-for-const.rs:2:11
+   |
+LL |     1_u32.MAX();
+   |     ------^^^--
+   |     |     |
+   |     |     this is an associated function, not a method
+   |     help: use associated function syntax instead: `u32::MAX()`
```

Comment:
We probably should filter out this suggestion... but I'll file a follow-up for someone who would like to dive deeper into method candidate assembly...